### PR TITLE
Wallet list

### PR DESCRIPTION
**Pull request by EmelyanenkoK**

### CODE REVIEW

**Comment by AAmir1361** [comment]
>

**Review by Volimjasnu12 — COMMENTED** [review]
Please help

**Review by Volimjasnu12 — COMMENTED** [review]
Please help

File: src/contract/wallet/WalletSources.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Wallets
+# Original Wallets
```
Comment (zbkash) [inline src/contract/wallet/WalletSources.md:1]: ****

**Review by Volimjasnu12 — COMMENTED** [review]
Best

File: src/contract/wallet/WalletSources.md (hunk 1 above)
Comment (ageloisacarter-beep) [reply src/contract/wallet/WalletSources.md:1]: Original

**Comment by ageloisacarter-beep** [comment]
0:1933aab6caba97c1679adbf81eed04359a72788665c8c94a6ddd840759656b90Originalhelpwallet_list

File: src/contract/wallet/WalletSources.md [hunk 2]
```diff
@@ -255,5 +255,79 @@ dup <s csr.
...
+
+
+# New wallets
+There are also a list of new wallets supported by TonWeb
+
+## V4 wallet
+### revision 1
+Is not used (and never was used in public).
+### revision 2
+Original repo https://github.com/ton-blockchain/wallet-contract
```
Comment (Kiflsh16) [inline src/contract/wallet/WalletSources.md]: How to back to the orginal

**Pull request closed without merging**